src/developer/web-api/tracker.md

```diff
@@ -1986,6 +2065,59 @@ Two endpoints are dedicated to events:
 - `GET /api/tracker/events/{id}`
     - retrieves an event given the provided id
 
+If not otherwise specified, JSON is the default response for the `GET` method. 
+The API also supports CSV export for single and collection endpoints. Furthermore, it supports 
+compressed JSON and CSV for the collection endpoint.
```

Comment:
There seem to be newlines in your sentences that I don't get like 

Furthermore, it supports 
compressed JSON and CSV for the collection endpoint. 

Why is there one after 'it supports'?

Reply:
ok sorry i didn't put the automatic return after 100